### DISCUSSION TITLE
Sending custom mail using NodeJS

### DISCUSSION THREAD

**Discussion by etfz** [pocketbase/pocketbase#5413] [Q&A]
Hi,

I'm looking to send custom mail. I've seen the documentation for this, but I'm not sure for example where I would import `MailerMessage` from.

**Comment by ganigeorgiev** [comment]
`MailerMessage` and any other PocketBase specific function or class are globally accessible, you don't have to import anything.

**Reply by ganigeorgiev** [reply]
Additionally note that the JS `pb_hooks` doesn't support ES modules, only CJS (see https://pocketbase.io/docs/js-overview/#loading-modules).

**Reply by etfz** [reply]
Sorry, I forgot to mention it's a NodeJS project, also. This is a TypeScript project that I've inherited, and I'm not that involved yet.

Edit: Ok, it's accessible via `global.MailerMessage`.

**Reply by ganigeorgiev** [reply]
I have no idea what do you mean, please keep in mind that the PocketBase JSVM is not a Node.js environment.

The JS hooks works only as part of the `pb_hooks` (and `pb_migrations`). For more details please read https://pocketbase.io/docs/js-overview/.

**Reply by etfz** [reply]
My bad. I may have gotten confused with the different environments. I am talking about the JavaScript SDK. I see that I probably need to use PocketBase hooks instead, though.
